### PR TITLE
Fix #40: Adjust section about DRBD/NFS

### LINKED ISSUE #40: DRBD/NFS Automated Configuration title in automation docs is not really accurate
The section title `DRBD/NFS Automated Configuration` in the automation documentation page is not really accurate.

The `drbd-formula` only configures DRBD, not NFS. In order to use NFS, we need to do things on top of it.

The thing is that SUSE recommends to use DRBD with HA to have a high available NFS share. For that, we configure the HA cluster in top of DRBD to create the NFS exports. But this is not anything the `drbd-formula` does.

Maybe the title is confusing.

If we want to document how to have high available NFS with DRBD, we will need to add more things (explain that we need to use `drbd-formula` and `habootstrap-formula`, and after that apply a specific configuration to the cluster to setup as NFS).

### PR DESCRIPTION
**Pull request by tomschr**
This PR contains:

* Shift focus towards NFS than DRBD
* Copy paragraph about habootstrap-formula
* Add links
* Stay consistent with the other sections

----

Looks like this:

Screenshot_20201028_114338

---

@arbulu89 What do you think? Does it fit better into the doc now? :wink:

### CODE REVIEW

File: xml/art_solution_automation.xml [hunk 1]
```diff
@@ -381,21 +381,33 @@ https://confluence.suse.com/x/8IEnGw
     </itemizedlist>
   </sect1>
 
-  <sect1 xml:id="sec-sol-autom-drbd-nfs-config">
-    <title>DRBD/NFS Automated Configuration</title>
+  <sect1 xml:id="sec-sol-autom-ha-nfs-config">
+    <title>&ha; NFS Storage</title>
```
Comment (arbulu89) [inline xml/art_solution_automation.xml:385]: Maybe better `High avaialble NFS Storage` rather than `High Availability NFS Storage`?

**Review by arbulu89 — APPROVED** [review]
I think it makes more sense this way. `NFS` is more meaningful for the users.

File: xml/art_solution_automation.xml (hunk 1 above)
Comment (tomschr) [reply xml/art_solution_automation.xml]: Yes, good idea. Integrated it in commit f10ba5d

Comment (tomschr) [reply xml/art_solution_automation.xml]: Anything else? Is it better now?

Comment (arbulu89) [reply xml/art_solution_automation.xml]: Yes. Thanks

**Comment by tomschr** [comment]
Also cherry-picked to

* `maintenance/15_GA: a784cc74ed9
* `maintenance/15_SP1: 2b25977
* `maintenance/15_SP2: e354c5b

**Pull request merged**